mem/data/test/DataMemWithCrossbarRTL_test.py

```diff
@@ -169,18 +171,18 @@ def test_const_queue(cmdline_opts):
   # Input data.
   # noc_send_read_addr = [AddrType(42)]
   send_to_noc_load_request_pkt = [
-             #   src  dst src_x src_y dst_x dst_y opq vc cmd                addr data predicate
-      NocPktType(0,   0,  0,    0,    0,    0,    0,  0, CMD_LOAD_REQUEST,  42,  0,   1),
+             #   src  dst src_x src_y dst_x dst_y tile_id opq vc cmd                addr data predicate
+      NocPktType(0, 0,   0,  0,    0,    0,    0, 0,      0,  0, CMD_LOAD_REQUEST,  42,  0,   1),
   ]
   noc_recv_load_data = [DataType(0xbbbb, 1)]
 
   # Expected.
   # noc_send_write_addr = [AddrType(40), AddrType(45)]
   # noc_send_write_data = [DataType(0xd040, 1), DataType(0xd545, 1)]
   send_to_noc_store_pkt = [
-             #   src  dst src_x src_y dst_x dst_y opq vc cmd                addr data    predicate
-      NocPktType(0,   0,  0,    0,    0,    0,    0,  0, CMD_STORE_REQUEST, 40,  0xd040, 1),
-      NocPktType(0,   0,  0,    0,    0,    0,    0,  0, CMD_STORE_REQUEST, 45,  0xd545, 1),
+             #   src  dst src_x src_y dst_x dst_y tile_id opq vc cmd                addr data    predicate

```

Comment:
Align these description with below values, and add the missing field descriptions.